.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,24 +7,28 @@ assignees: ''
 
 ---
 
-Please take the time to describe the bug in detail, so other contributors can pick it up and implement or declare done/obsolete in case it was fixed out of band.
+Please take the time to describe the issue in detail, so that other contributors can pick it up and implement or declare
+it done/obsolete in case it was fixed out of band.
 
-**Current behaviour**
+**Current behaviour/situation**
 * Describe what is wrong with the system as of writing: what is the actual state of things.
-* Instructions to reproduce the bug.
-* Screenshots, screencasts, or video to help explain your problem.
+* Instructions to reproduce the problem.
+* Screenshots, screencasts, photos or video to help explain your problem.
 
-**Expected behaviour**
+**Expected state/behaviour**
 * Describe what is the expected state of things.
-* Identify documents that need to be updated, especially in case of hardware component choices or changes assembly.
+* Identify documents that need to be updated, especially in case of hardware component choices or changes in assembly.
 
 **How do you know it is wrong**
 
-Link any documents, Slack threads, quote personal communication, reference your personal domain expertise etc. explaining why you think the current state of things is bad and the expected state of things is good.
+Link any documents, Slack threads, quote personal communication, reference your personal domain expertise etc.
+explaining why you think the current state of things is bad and the expected state of things is good.
 
-**Code pointers**
+Link to requirements or risk analysis if applicable.
 
-Give some thoughts on roughly where in code do you think the necessary changes might be located.
+**Starting points**
+Where might one start with solving this problem? Point to any relevant files or websites. Even if you have but a vague
+idea, please share. Perhaps something has prevented you or blocked you from solving the problem already?
 
 **Don't forget**
 - [ ] Put the issue in a milestone
```

Comment:
Do we want the author of the ticket to decide the milestone? Seems like the area lead should assign an issue to a milestone.

Reply:
To me, I'd think if someone in making a ticket, they probably have an idea of what they are trying to achieve and therefore what milestone that would fall under

Reply:
Mostly the problem has been things not being categorized by anyone, because we lack consistent project manager roles. So the idea was that anyone who can do it should do it. Essentially, only members of the organization should be able to do this anyways, so if a ticket is being filed by an outsider, they'll be blocked from doing it anyway.